### PR TITLE
os/drivers/input: Add ioctl for Touch Wakeup and suspend

### PR DESCRIPTION
**Pull request by namanjain7**
Add support for Touch wakeup and touch suspend. Added ioctl for it

### CODE REVIEW

File: os/include/tinyara/input/touchscreen.h [hunk 1]
```diff
@@ -84,6 +84,9 @@
 #define TSIOC_SETFREQUENCY   _TSIOC(0x0003)  /* arg: Pointer to uint32_t frequency value */
 #define TSIOC_GETFREQUENCY   _TSIOC(0x0004)  /* arg: Pointer to uint32_t frequency value */
 
+#define TSIOC_SUSPEND        _TSIOC(0x0006)  /* Disable touch interrupts */
+#define TSIOC_WAKEUP         _TSIOC(0x0007)  /* Enable touch interrupts */
```
Comment (sunghan-chang) [inline os/include/tinyara/input/touchscreen.h:88]: interrupt

Let's remove `s`

**Review by sunghan-chang — COMMENTED** [review]
Let's add location `input` at the title
```
os/drivers/input: Add ....
```

**Comment by sunghan-chang** [comment]
@ewoodev Could you review this?

File: os/include/tinyara/input/touchscreen.h (hunk 1 above)
Comment (namanjain7) [reply os/include/tinyara/input/touchscreen.h:88]: done

File: os/drivers/input/touchscreen.c [hunk 2]
```diff
@@ -229,7 +229,14 @@ static int touch_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
...
-		default:
+		case TSIOC_SUSPEND: {
+			priv->ops->touch_disable(priv);
+		}
+		break;
+
+		case TSIOC_WAKEUP: {
+			priv->ops->touch_enable(priv);
+		}
 		break;
```
Comment (sunghan-chang) [inline os/drivers/input/touchscreen.c:240]: Sorry but real operations are touch disable and enable. But the commands are SUSPEND and WAKEUP. wakeup and enable look little bit different. Is it impossible to use ENABLE and DISABLE?

**Comment by namanjain7** [comment]
done all changes

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**